templates/partner/intel.html

```diff
@@ -0,0 +1,645 @@
+            <a href="/blog/profile-workloads-on-x86-64-v3-to-enable-future-performance-gains">Desktops&nbsp;&rsaquo;</a>
+          </h3>
+        </div>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item u-hide--small u-hide--medium">
+          <div class="p-image-container">
+            {{ image(url="https://assets.ubuntu.com/v1/a6b2c43b-servers.png",
+                        alt="Servers",
+                        width="852",
+                        height="1278",
+                        hi_def=True,
+                        loading="auto|lazy",
+                        attrs={"class": "p-image-container__image"},) | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">
+            <a href="/blog/optimised-real-time-ubuntu-is-now-generally-available-on-intel-socs">Servers&nbsp;&rsaquo;</a>
+          </h3>
+        </div>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item u-hide--small u-hide--medium">
+          <div class="p-image-container">
+            {{ image(url="https://assets.ubuntu.com/v1/04f42dc3-iot-devices.png",
+                        alt="IoT devices",
+                        width="852",
+                        height="1278",
+                        hi_def=True,
+                        attrs={"class": "p-image-container__image"},
+                        loading="auto|lazy") | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">
+            <a href="/blog/ubuntu-performance-engineering-with-frame-pointers-by-default">IoT devices&nbsp;&rsaquo;</a>
+          </h3>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section--deep">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Want to optimize your silicon for Ubuntu?</h2>
+      </div>
+      <div class="col">
+        <p>
+          Partnering with Canonical empowers developers with the ability to innovate more rapidly on your platforms and extract the unique value of your hardware, all while providing enterprises with the peace of mind that comes with reliable long term support.
+        </p>
+        <hr class="p-rule--muted" />
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Accelerated time to market</li>
+          <li class="p-list__item is-ticked">Easy-to-deploy optimized Ubuntu images</li>
+          <li class="p-list__item is-ticked">Custom feature additions and optimizations</li>
+          <li class="p-list__item is-ticked">Security maintenance for 3rd party open source</li>
+          <li class="p-list__item is-ticked">Joint marketing and co-selling</li>
+          <li class="p-list__item is-ticked">Access to the Ubuntu developer ecosystem</li>
```

Comment:
On design, the divided list section is wrapped in shallow section
<img width="883" alt="Screenshot 2024-12-18 at 3 47 11 PM" src="https://github.com/user-attachments/assets/6a893ff4-c06b-488d-a24e-ad5139b05ab7" />

Reply:
![image](https://github.com/user-attachments/assets/a7daa0a3-f9eb-464a-91df-a0d32c33689f)
There is already padding from the list